.github/workflows/main.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu, macos]
-        ruby: [2.5, 2.6, 2.7, head, debug, truffleruby, truffleruby-head]
+        ruby: [2.5, 2.6, 2.7, '3.0', head, debug, truffleruby, truffleruby-head]
```

Comment:
> we have to use quotes for '3.0'

-- https://github.com/ruby/setup-ruby/blob/b0055144/README.md#matrix-of-ruby-versions